main.go

```diff
@@ -33,14 +33,21 @@ func whoAmI() string {
 }
 
 func main() {
+	var BuildVersion string
 	log.SetFlags(log.LstdFlags | log.Lshortfile)
 	port := flag.Int("p", 3128, "port number to listen on")
 	pacURLFromFlag := flag.String("C", "", "url of proxy auto-config (pac) file")
 	domain := flag.String("d", "", "domain of the proxy account (for NTLM auth)")
+	version := flag.Bool("v", false, "print version number")
```

Comment:
To me, when I see -v I usually think "verbose". What do you think about using -version instead?

Also, I'm nitpicking here, but to me domain and username go together, so it's looks strange that you've put version in between them. Maybe order the variables by the order that we use them, and define version just above port?

Reply:
I hear you the problem is -v is usually short for -version.

Reply:
https://github.com/samuong/alpaca/pull/81/commits/e22f97913e754750cfd396ddc637b0de71ffe338